index.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+  -->
+  <div class="sidebar">
+    <nav id="navbar">
+        <header class="title">
+          <p>JS Documentation</p>
+        </header>
+          <a class="nav-link" href="#Introduction">Introduction</a>
+          <a class="nav-link" href="#What_you_should_already_know">What you should already know</a>
+          <a class="nav-link" href="#JavaScript_and_Java">JavaScript and java</a>
+          <a class="nav-link" href="#Hello_world">Hello World</a>
+          <a class="nav-link" href="#Variables">Variables</a>
+          <a class="nav-link" href="#Declaring_variables">Declaring variables</a>
+          <a class="nav-link" href="#Variable_scope">Variable Scope</a>
+          <a class="nav-link" href="#Global_variables">Global variables</a>
+          <a class="nav-link" href="#Constants">Constants</a>
+          <a class="nav-link" href="#Data_types">Data types</a>
+          <a class="nav-link" href="#if...else_statement">if...else statement</a>
+          <a class="nav-link" href="#while_statement">While statement</a>
+          <a class="nav-link" href="#Function_declarations">Function declarations</a>
+          <a class="nav-link" href="#Reference">Reference</a>
+    </nav>
+  </div>
+
+  <!--Main Page-->
+  <main id="main-doc">
+  <section class="main-section" id="Introduction">
+    <header>Introduction</header>
```

Comment:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear. eg why is having only one ```<h1>``` in your project recommended